arbstate/inbox.go

```diff
@@ -147,10 +146,17 @@ func ParseSequencerMessage(ctx context.Context, batchNum uint64, batchBlockHash
 
 	// At this point, `payload` has not been validated by the sequencer inbox at all.
 	// It's not safe to trust any part of the payload from this point onwards.
+	var uncompressedBatchSizeLimit uint64
+	if chainConfig != nil {
+		uncompressedBatchSizeLimit = chainConfig.MaxUncompressedBatchSize()
```

Comment:
I went back-and-forth on it, but I think I landed on the side of:
We should make it arbos-version dependant, so:
* MaxUncompressedBatchSize should accept arbosVersion as parameter
* should only use non-default size if arbos-version is > (some new constant like arbosVersion_BatchSize that'll be initially set to 52)

Reply:
- I updated the geth PR
- I adjusted the nitro code by passing `ArbosVersionGetter` between some components - unfortunately this was the most concise way I could come up with